internal/controller/servicemgr/executor/executor_test.go

```diff
@@ -1 +1,30 @@
 package executor
+
```

Comment:
Seems that we need to put the copyright statement and license at the top of this file. Thought? 😁